arbeitszeit_flask/templates/company/my_accounts.html

```diff
@@ -60,7 +60,8 @@ <h1 class="title">
 
                     </tr>
                     <tr>
-                        <td class="py-2 has-text-left">Account prd</td>
+                        <td class="py-2 has-text-left"><a href="{{ url_for('main_company.account_prd') }}">Account
```

Comment:
"Account prd" should be made translatable.

Reply:
in this file are more strings that should be made translatable, i make them all translatable.